templates/services_user_data.tpl

```diff
@@ -34,6 +34,51 @@ apt-get update
 apt-get install -y "linux-image-$UNAME"
 apt-get -y install docker-ce="17.12.1~ce-0~ubuntu"
 
+##### fluentd configuration #####
+curl -L https://toolbelt.treasuredata.com/sh/install-ubuntu-xenial-td-agent2.5.sh | sh
```

Comment:
i thought we have our own docker image ? could we not use that ?

Reply:
No, fluentd is not running in docker, its running as a daemon the services node to enable default docker logging driver to fluentd